data/transit/route/bus.json

```diff
@@ -2084,13 +2095,7 @@
       }
     },
     {
-      "displayName": "CAT",
```

Comment:
If you just remove something without match it's going to come back when the data gets generated again.